shared/data/bundled/extension-config.json

```diff
@@ -1,6 +1,6 @@
 {
     "readme": "https://github.com/duckduckgo/privacy-configuration",
-    "version": 1642335495631,
+    "version": 1643197961938,
```

Comment:
IMO we should not mix config updates with unrelated PRs, but rather have one PR that updates the config just before the release.

Reply:
I think we'd want this tbf as the key has changed. We could probably mark this file as a binary as it's not really that meaningful anyway.

Reply:
Agreed. Using our current flow for testing config changes this updated the embedded config when I removed the test version. When we're ready to ship we'll use the version that includes this feature.